webpack/components/extensions/HostDetails/DetailsTabCards/HwPropertiesCard.js

```diff
@@ -43,7 +44,7 @@ const HwPropertiesCard = ({ isExpandedGlobal, hostDetails }) => {
   const coreSocket = facts?.['cpu::core(s)_per_socket'];
```

Comment:
I think this is `reportedFacts?.cores`. I can't comment on it, but above there's also `reportedFacts?.sockets`